### ISSUE TITLE
QSTNet/experiments/bat/rgbt.yaml中参数问题

### ISSUE THREAD

**Issue by Guobinyuan** [dongdong2061/QSTNet#5]
QSTNet/experiments/bat/rgbt.yaml中的TEST部分，有EPOCH: 60参数，该参数是否代表推理阶段所使用的是第60轮的训练结果，如果是的话，是否应该修改为15？我参考了BAT和ViPT的工作，都是训练60轮，测试使用第60轮，而贵工作训练15轮次，或许应该使用第15轮的模型进行推理

**Comment by dongdong2061** [comment]
Both BAT and ViPT adopt a partial fine-tuning strategy, whereas we employ full fine-tuning. Therefore, we train the model for only 15 epochs and report the results obtained at the 15th epoch.

**Comment by Guobinyuan** [comment]
If I were to use your code, should I modify it to 15?

**Comment by dongdong2061** [comment]
No. In this implementation ([https://github.com/dongdong2061/QSTNet/blob/main/lib/test/parameter/bat.py](https://github.com/dongdong2061/QSTNet/blob/main/lib/test/parameter/bat.py)), the number of testing epochs is already fixed to 15.

**Comment by Guobinyuan** [comment]
Thank you，really！

**Comment by Guobinyuan** [comment]
But when I run the test with the command CUDA_VISIBLE_DEVICES=2 NCCL_P2P_LEVEL=NVL python ./RGBT_workspace/test_rgbt_mgpus.py --script_name bat --dataset_name LasHeR --yaml_name rgbt, the output is:——————————Process sequence: baggirl——————————————baggirl , fps:4.885241737631108test config: {'MODEL': {'PRETRAIN_FILE': './pretrained/OSTrack_ep0300.pth.tar', 'EXTRA_MERGER': False, 'RETURN_INTER': False, 'RETURN_STAGES': [], 'BACKBONE': {'TYPE': 'vit_base_patch16_224_ce_adapter', 'STRIDE': 16, 'MID_PE': False, 'SEP_SEG': False, 'CAT_MODE': 'direct', 'MERGE_LAYER': 0, 'ADD_CLS_TOKEN': False, 'CLS_TOKEN_USE_MODE': 'ignore', 'CE_LOC': [3, 6, 9], 'CE_KEEP_RATIO': [1, 1, 1], 'CE_TEMPLATE_RANGE': 'CTR_POINT'}, 'HEAD': {'TYPE': 'CENTER', 'NUM_CHANNELS': 256}}, 'TRAIN': {'PROMPT': {'TYPE': 'TDMF'}, 'LR': 0.0001, 'WEIGHT_DECAY': 0.0001, 'EPOCH': 15, 'LR_DROP_EPOCH': 12, 'BATCH_SIZE': 8, 'NUM_WORKER': 10, 'OPTIMIZER': 'ADAMW', 'BACKBONE_MULTIPLIER': 0.1, 'GIOU_WEIGHT': 2.0, 'L1_WEIGHT': 5.0, 'FREEZE_LAYERS': [0], 'PRINT_INTERVAL': 50, 'VAL_EPOCH_INTERVAL': 1, 'GRAD_CLIP_NORM': 0.1, 'AMP': False, 'FIX_BN': False, 'SAVE_EPOCH_INTERVAL': 2, 'SAVE_LAST_N_EPOCH': 1, 'CE_START_EPOCH': 4, 'CE_WARM_EPOCH': 16, 'DROP_PATH_RATE': 0.1, 'SCHEDULER': {'TYPE': 'step', 'DECAY_RATE': 0.1}}, 'DATA': {'SAMPLER_MODE': 'causal', 'MEAN': [0.485, 0.456, 0.406], 'STD': [0.229, 0.224, 0.225], 'MAX_SAMPLE_INTERVAL': 200, 'TRAIN': {'DATASETS_NAME': ['LasHeR_train'], 'DATASETS_RATIO': [1], 'SAMPLE_PER_EPOCH': 60000}, 'VAL': {'DATASETS_NAME': ['LasHeR_val'], 'DATASETS_RATIO': [1], 'SAMPLE_PER_EPOCH': 10000}, 'SEARCH': {'SIZE': 256, 'FACTOR': 4.0, 'CENTER_JITTER': 3, 'SCALE_JITTER': 0.25, 'NUMBER': 1}, 'TEMPLATE': {'NUMBER': 1, 'SIZE': 128, 'FACTOR': 2.0, 'CENTER_JITTER': 0, 'SCALE_JITTER': 0}}, 'TEST': {'TEMPLATE_FACTOR': 2.0, 'TEMPLATE_SIZE': 128, 'SEARCH_FACTOR': 4.0, 'SEARCH_SIZE': 256, 'EPOCH': 60}}
Does this indicate that there is an error in my configuration, and the trained model tta/code/QSTNet/output/checkpoints/train/bat/rgbt/BATrack_ep0015.pth.tar is not being used correctly? If so, how should I handle it?

**Comment by dongdong2061** [comment]
fps: 4.885241737631108, which indicates that the test was correctly conducted with no issues.




秋冬晚归客
***@***.***

**Comment by Guobinyuan** [comment]
I see now, and I would like to express my gratitude again for your help!
